crates/ruff_linter/resources/test/fixtures/pycodestyle/E20.py

```diff
@@ -89,3 +89,49 @@
 f"{ {'a': 1} }"
 f"{[ { {'a': 1} } ]}"
 f"normal { {f"{ { [1, 2] } }" } } normal"
+
+#: Okay
+ham[lower + offset : upper + offset]
+
+#: Okay
+ham[(lower + offset) : upper + offset]
+
+#: E203:1:19
+ham{lower + offset : upper + offset}
+
+#: E203:1:19
+ham[lower + offset  : upper + offset]
+
+#: Okay
+release_lines = history_file_lines[history_file_lines.index('## Unreleased') + 1: -1]
```

Comment:
Wouldn't this now be not okay, since the spacing is different on the other side?

Reply:
The intent was to allow _either_ this or the variant with the space.

Reply:
Oh okay, interesting choice